specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewayConnectionStartPacketCapture.json

```diff
@@ -0,0 +1,39 @@
+{
+	"example1":{
```

Comment:
If you have two examples, please split them into two files (you can add as many examples as you want to `x-ms-examples`). This syntax is wrong.

Reply:
Okay I will change this.

Reply:
Changed. Let me know if I need to change anything else.